scania-digital-design-system/sdds
# Feature - on white bg + some more updates

**Describe pull-request**  
_Describe what the pull-request is about_

Add the possibility to change within the light theme background of the component based on the background it is put on. 

You can add a class called .sdds-on-white-bg and it will switch the css value of the background color depending on the component

- Textfield
- Accordion (hover)
- Dropdown
- Card box-shadow changed

Other stuff in this branch
- Update angular example with latest packages and some small changes in the HTML
- Added missing components in example(modal, sdds-icon)
- Fix icon warning that is shown when using sdds-icon
- Updated react demo  also

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #68 [AB#365](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/365)

**How to test**  
_Add description how to test if possible_
1. Go to storybook preview or run storybook locally 
2. On the top of the navbar in storybook a new icon is added that looks the same as changing background. One button is for changing the background color and one is changing the background of the component. Try both. **Screenshot added**
3. Check all the components mentioned

**Screenshots**  
_If applicable, add screenshots to help explain_
![image](https://user-images.githubusercontent.com/35451568/119843583-fae63800-bf07-11eb-9a18-6818540f8ca0.png)


**Additional context**  
_Add any other context about the pull-request here._

Updating the other examples also, but if this gets merged before it is fine. I will create a another PR for the rest of the demos
